src/Makefile.test.include

```diff
@@ -374,7 +374,7 @@ endif
 
 if ENABLE_TESTS
 UNIVALUE_TESTS = univalue/test/object univalue/test/unitester
-noinst_PROGRAMS += $(UNIVALUE_TESTS)
+check_PROGRAMS += $(UNIVALUE_TESTS)
```

Comment:
@MarcoFalke Yes. it is. Why did you point at it?

Reply:
Why?

`test_bitcoin` is a part of the distributed package, but univalue tests are not. Why build them in `contrib/guix/libexec/build.sh`?

Reply:
I wasn't aware this changes the guix build and the release tarball. Maybe mention it in the description?

Reply:
This PR does not change the release tarball.

Reply:
I find it convenient to be notified of univalue build failures on `make` when developing univalue.

Reply:
"univalue build failures" or "univalue tests build failures"?

Reply:
Both.

Did you measure how many milliseconds this shaves off of the guix build? I'd bet it is far less than 0.1% of overall build time.

Unrelated: It may overall be more useful for devs to introduce a `make check_quick` to only run the bitcoin core unit tests (without any secp exhaustive_tests)

Reply:
Thanks! [Updated](https://github.com/bitcoin/bitcoin/pull/23670#issuecomment-1191281796).